app/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```

Comment:
Fdroid does not support SDK 30 yet. To avoid failing builds I'd wait till fdroid updates

Reply:
Oh i didn't know that

Reply:
You could remove the sdk bump to get it merged right away or I'll check in here again once sdk 30 is available.